.github/workflows/release-please.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+      - develop
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v3.0.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this still necessary? Or left over from testing?

Reply:
I want to leave it in while confirming that the same config works here as on my fork, but can remove it after I confirm functionality.